app/services/site_channel_domain_setter.rb

```diff
@@ -85,4 +85,10 @@ def inspect_host
 
   class DomainExclusionError < RuntimeError
   end
+
+  private
+
+  def site_already_verified?(channel_details)
+    SiteChannelDetails.joins(:channel).where(brave_publisher_id: channel_details.brave_publisher_id, "channels.verified": true).any?
```

Comment:
Would this be simplified to: `channel_details.channel.verified?`?

Reply:
This method should return true if any verified site has the same brave publisher id, so not the same.  But this code will be removed anyway since we're allowing publishers to try and verify a site that has already been verified.